src/telemetry/Manager.cc

```diff
@@ -38,6 +43,232 @@ void Manager::InitPostBrokerSetup(broker::endpoint& ep)
 	pimpl.swap(ptr);
 	}
 
+// -- collect metric stuff -----------------------------------------------------
+
+const std::vector<std::string_view> Manager::Metric::label_names() const
+	{
+	std::vector<std::string_view> r;
+
+	// Labels are backed by strings, so string_view should be safe.
+	for ( auto const& l : broker::telemetry::label_names(family_) )
+		r.push_back(l);
+
+	return r;
+	}
+
+Manager::Metric::Metric(Manager::Metric::MetricType metric_type,
+                        const broker::telemetry::metric_family_hdl* family,
+                        std::optional<double> value, std::optional<int64_t> count_value,
+                        std::vector<std::string_view> labels)
```

Comment:
Any reason not to use the same order for ctor arguments as in class and initialization? Generally, less confusing if all is the same order.

Reply:
Uhm, certainly not intentional. Thanks.